### ISSUE TITLE
Failed to parse OpenAPI YML with x-* custom properties

### ISSUE THREAD

**Issue by jethrolam** [kiwicom/schemathesis#448]
We have a few OpenAPI YMLs that make use of the x-* custom properties:
https://swagger.io/docs/specification/openapi-extensions/

In our case, we have a `x-handler` under each path:

```
paths:
  /tables/init:
    x-handler: foobar
    get:
      tags:
        - foorbar
      ...
```

The error we received:
```
Traceback (most recent call last):
  File "/var/lang/lib/python3.7/site-packages/schemathesis/schemas.py", line 210, in get_all_endpoints
    or should_skip_by_tag(definition.get("tags"), self.tag)
AttributeError: 'str' object has no attribute 'get'
```

So it looks like it attempts to parse `x-handler` like it does to GET and POST.

**Comment by Stranger6667** [comment]
Hi @jethrolam !
Thank you very much for the report, I will fix it soon

**Comment by Stranger6667** [comment]
@jethrolam 
The issue should be resolved in `0.28.0` (https://pypi.org/project/schemathesis/0.28.0/)

Let me know if it won't fix the error you reported and feel free to reopen this issue in that case.
